mmv1/products/compute/RegionTargetHttpsProxy.yaml

```
@@ -140,8 +140,7 @@ properties:
     name: 'certificateManagerCertificates'
     description: |
       URLs to certificate manager certificate resources that are used to authenticate connections between users and the load balancer.
-      Currently, you may specify up to 15 certificates. Certificate manager certificates do not apply when the load balancing scheme is set to INTERNAL_SELF_MANAGED.
-      sslCertificates and certificateManagerCertificates fields can not be defined together.
+      sslCertificates and certificateManagerCertificates can not be defined together.
```

https://developers.google.com/style/contractions#negation-contractions

```suggestion
      sslCertificates and certificateManagerCertificates can't be defined together.
```

---

Done!